### PR TITLE
CRUD - Delete a single tutorial

### PR DESCRIPTION
**Pull request by naoharu**

### CODE REVIEW

**Comment by naoharu** [comment]
# initial data
image

# delete
image

# result
image

**Comment by naoharu** [comment]
Hi @toshihue - Could you review this PR?

**Comment by toshihue** [comment]
> Hi @toshihue - Could you review this PR?

@naoharu Thanks! I have confirmed deleting a single tutorial document by the ObjectId is successful.

File: mongodb/init.js
Comment (toshihue) [inline mongodb/init.js:1]: Hading over the ObjectIds to the front-end side needs to be considered later.

**Review by toshihue — APPROVED** [review]

**Pull request merged**